backend/cmd/migrate/migrations/000004_experiments_schema_changes.up.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS experiments;
+CREATE TABLE IF NOT EXISTS experiment_config (
+    id BIGINT,
+    details JSONB
+);
+CREATE TABLE IF NOT EXISTS experiment_run (
+    id BIGINT,
+    experiment_config_id BIGINT,
+    start_time TIMESTAMP WITH TIME ZONE,
```

Comment:
How were you planning to calculate the start_time? This would involve an update to this row whenever the experiment is actually started vs. created. Do you expect this to be that different enough to provide value?

Reply:
right now start and create happen in the same query, at least for FI. When we support scheduling, create will be NOW() and start will be some time in the future.